python-package/xgboost/__init__.py

```diff
@@ -12,6 +12,7 @@
 from . import tracker  # noqa
 from .tracker import RabitTracker  # noqa
 from . import dask
+from .federated import run_federated_server
```

Comment:
Let's avoid importing this for now.  I thought we might have a 3 party implementation of server like nvflare?

Reply:
Done.